flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionMySQLSink.java

```diff
@@ -66,7 +66,7 @@
      */
     public OrionMySQLSink() {
         super();
-        logger = CygnusLogger.getLogger(OrionHDFSSink.class);
+        logger = CygnusLogger.getLogger(OrionMySQLSink.class);
```

Comment:
The SQL sink was logging as the HDFS sink... nothing critical. Anyway, I'll made a fix in release/0.6.0 before tagging to release/iotplatform-v1.1